docs/features/kubernetes/configuration.md

```diff
@@ -136,6 +138,35 @@ kubectl -n <NAMESPACE> get secret $(kubectl -n <NAMESPACE> get sa <SERVICE_ACCOU
 | base64 --decode
 ```
 
+For Kubernetes 1.24+, as described in [this
+guide](https://kubernetes.io/docs/concepts/configuration/secret/#service-account-token-secrets),
+you can obtain a long-lived token by creating a secret:
+
+```sh
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <SECRET_NAME>
+  namespace: <NAMESPACE>
+  annotations:
+    kubernetes.io/service-account.name: <SERVICE_ACCOUNT_NAME>
+type: kubernetes.io/service-account-token
+EOF
+```
+
+waiting for the token controller to populate a token, and retrieving it with:
+
+```sh
+kubectl -n <NAMESPACE> get secret <SECRET_NAME> -o go-template='{{.data.token | base64decode}}'
+```
```

Comment:
It seems weird to recommend using a long-lived, manually created service account token, instead of making use of the normal short-lived authentication baked into Kubernetes.
For running Backstage in-cluster, I would move this to a FAQ for custom installations of Backstage, and recommend omitting the `serviceAccountToken` field.

Reply:
Hmmm, I wouldn't say that using `serviceAccount` at all is actually _recommended_ -- probably something federated like `google` or `aks` or `oidc` is a better practice -- maybe we need to explicitly mention that using long-lived service account tokens is not ideal?

If Backstage isn't running in-cluster (or isn't deployed on k8s at all) then I think there is no "normal" authentication, but again, maybe if we put a proper disclaimer here ("Yeah we know that long-lived credentials like this are a bad idea, that's a known weakness of the `serviceAccount` auth provider") that would alleviate the weirdness you're perceiving?

Or maybe you would rather see an instruction that mentions something like `kubectl create token` and leaves credential rotation as an exercise to the reader?

Reply:
Agree that using service account auth is not ideal but I think it is good to support as it is a form of auth that will always work with any k8s setup, even bare metal with nothing installed on it.

Reply:
@AndersBennedsgaard I hope 26fde684d6152f453b675de5c417a78828a9ca69 does justice to the spirit of your remarks. If I'm being honest, I'm still not confident I properly appreciated the "FAQ" comment.